src/components/helpers/RegMenuHelpers.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+// helper
+const getCallerSaved = (registers) => (
+  registers.filter((reg) => reg.saver === 'Caller').map((reg) => (
+    <li key={reg.name}>
+      { reg.name }
```

Comment:
what about using a template string? ```
`${reg.name} (${reg.abiName}): ${reg.value}`
```

Reply:
tried this, eslint still gives the complaint about how each element needs to be on its own line

Reply:
even if you do
```
<ReactElement>
  {`${reg.name} (${reg.abiName}): ${reg.value}`}
</ReactElement>
```

?

I don't think ESLint would complain about that. but if there's no other way around it then sure